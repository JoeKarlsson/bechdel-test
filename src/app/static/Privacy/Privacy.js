import React from 'react';

const Privacy = () => (
	<div className="container">
		<h1>privacy policy</h1>
		<p>
			If you require any more information or have any questions about our
			privacy policy, please feel free to contact us by email at webmaster at
			this domain.
		</p>
		<p>
			At bechdel.io, the privacy of our visitors is of extreme importance to us.
			This privacy policy document outlines the types of personal information is
			received and collected by bechdel.io and how it is used.
		</p>
		<h2>log files</h2>
		<p>
			Like many other Web sites, bechdel.io makes use of log files. The
			information inside the log files includes internet protocol (IP)
			addresses, type of browser, Internet Service Provider (ISP), date/time
			stamp, referring/exit pages, and number of clicks to analyze trends,
			administer the site, track user&apos;s movement around the site, and
			gather demographic information. IP addresses, and other such information
			are not linked to any information that is personally identifiable.
		</p>
		<h2>cookies</h2>
		<p>bechdel.io does not use cookies.</p>
		<h2>DoubleClick DART cookie</h2>
		<p>
			<ul>
				<li>
					Google, as a third party vendor, uses cookies to serve ads on
					bechdel.io.
				</li>
				<li>
					Google&apos;s use of the DART cookie enables it to serve ads to users
					based on their visit to bechdel.io and other sites on the Internet.
				</li>
				<li>
					Users may opt out of the use of the DART cookie by visiting the Google
					ad and content network privacy policy at the following URL -
					http://www.google.com/privacy_ads.html
				</li>
			</ul>
		</p>
		<p>
			Our advertising partner Google Adsense may use cookies and web beacons on
			our site.
		</p>

		<p>
			These third-party ad servers or ad networks use technology to the
			advertisements and links that appear on bechdel.io send directly to your
			browsers. They automatically receive your IP address when this occurs.
			Other technologies (such as cookies, JavaScript, or Web Beacons) may also
			be used by the third-party ad networks to measure the effectiveness of
			their advertisements and / or to personalize the advertising content that
			you see.
		</p>

		<p>
			bechdel.io has no access to or control over these cookies that are used by
			third-party advertisers.
		</p>

		<p>
			You should consult the respective privacy policies of these third-party ad
			servers for more detailed information on their practices as well as for
			instructions about how to opt-out of certain practices. bechdel.io&apos;s
			privacy policy does not apply to, and we cannot control the activities of,
			such other advertisers or web sites.
		</p>

		<p>
			If you wish to disable cookies, you may do so through your individual
			browser options. More detailed information about cookie management with
			specific web browsers can be found at the browsers&apos;respective
			websites.
		</p>
	</div>
);

export default Privacy;
