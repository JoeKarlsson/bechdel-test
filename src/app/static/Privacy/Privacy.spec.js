import React from 'react';
import Privacy from './Privacy';

describe('Privacy Page', () => {
	describe('rendering', () => {
		it('match the snapshot', () => {
			const wrapper = shallow(<Privacy />);
			expect(wrapper).toMatchSnapshot();
		});
	});
});
