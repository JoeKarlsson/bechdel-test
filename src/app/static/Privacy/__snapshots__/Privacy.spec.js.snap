// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Privacy Page rendering match the snapshot 1`] = `ShallowWrapper {}`;
